### PR TITLE
Sierra: Add translation prefix support for ILS messages.

### PR DESCRIPTION
**Pull request by EreMaijala**
Very much alike the one in Alma.

### CODE REVIEW

File: config/vufind/SierraRest.ini [hunk 1]
```diff
@@ -36,6 +36,14 @@ use_prefixed_ids = false
 ; Default is 2. Set to 0 to disable retries.
 ;http_retries = 2
 
+; Translation prefix to use for messages coming from Sierra. By default a prefix is
+; not used, but a prefix may be used to distinguish the codes from any other
+; translations (or other libraries). To use a simple prefix in the default text
+; domain:
+; translationPrefix = "sierra_"
```
Comment (demiankatz) [inline config/vufind/SierraRest.ini]: The comments are a little confusing here; you have text explaining the first example, but no text explaining the second example... and then the "actual" commented-out setting is identical to the second example. How would you feel about eliminating the two example lines, and changing the last sentence to something like the following?

You can use a simple prefix in the default domain (e.g. "sierra_") or a custom text domain (e.g. "ILSMessages::").

**Review by demiankatz — CHANGES_REQUESTED** [review]
Looks good to me, apart from a minor complaint about the comments.

File: config/vufind/SierraRest.ini (hunk 1 above)
Comment (EreMaijala) [reply config/vufind/SierraRest.ini]: It was just copied from Alma.ini, but I'll revise.

Comment (EreMaijala) [reply config/vufind/SierraRest.ini]: Should be better now.

**Review by demiankatz — APPROVED** [review]
Thanks, @EreMaijala!

**Pull request merged**